R/workflow.R

```diff
@@ -15,6 +16,7 @@ Workflow <- R6::R6Class(
     taskNames = NULL,
     reportFileName = NULL,
     createWordReport = NULL,
+    userDefinedTask = NULL,

```

Comment:
Should not we provide a **list** of user defined tasks instead? Or what is the reason to restricting userdef tasks to only **one**?

Reply:
The reason was during the run of the workflow, I am calling the plot as `self[[taskName]]$run` if userDefinedTask is a list it will crash.
What I can do, is to run the user defined tasks as a loop distinct from the plot tasks, so this slot in the workflow environment can become a list of tasks and won't be limited to only one task.
Would this work better ?

Reply:
Yes, I think it would be better